src/ZoomLoop.App/src/app/components/dashboard-card/dashboard-card.component.ts

```diff
@@ -0,0 +1,18 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'zl-dashboard-card',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './dashboard-card.component.html',
```

Comment:
Remove the component suffix in files and class name.

Reply:
The typescript name of the component should be Dashboard not DashboardComponent. The file name should be dashboard.ts not dashboard.component.ts